definitions/variable/Electricitytags_add.yaml

```
@@ -0,0 +1,41 @@
+
+- Electricity Source:
+    - Gas|CCGT|w/ CCS:
+        description: Combined cycle gas turbine with carbon capture and storage
```

Please start all descriptions in tag-lists with lower-case so that combined descriptions (variable plus tag-description) looks like one sentence.
```suggestion
        description: combined cycle gas turbine with carbon capture and storage
```